cmd/sonobuoy/app/logs.go

```diff
@@ -31,15 +35,55 @@ func init() {
 		Short: "Dumps the logs of the currently running sonobuoy containers for diagnostics",
 		Run:   getLogs,
 	}
-	// TODO: Determine options
+	cmd.Flags().BoolP("follow", "f", false, "Specify if the logs should be streamed.")
 	RootCmd.AddCommand(cmd)
 }
 
 func getLogs(cmd *cobra.Command, args []string) {
-	code := 0
-	if err := ops.GetLogs( /*opts*/ ); err != nil {
+	clientset, err := getClientset()
+	if err != nil {
+		errlog.LogError(errors.Wrap(err, "failed to get clientset"))
+		os.Exit(1)
+	}
+
+	follow, err := cmd.Flags().GetBool("follow")
+	if err != nil {
+		errlog.LogError(errors.Wrap(err, "error getting follow flag"))
+		os.Exit(1)
+	}
+
+	logOptions := &ops.LogOptions{
+		Follow: follow,
```

Comment:
If the number of options over time still = 1 then we just flatten this.

Reply:
flattened because we can expand it later.